src/styles/card.scss

```
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+.cards {
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 488px) {
+  .cards {
+    --column-count: 1;
+  }
+}
+
+@media (min-width: 488px) {
+  .cards {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    --column-count: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 36px;
+  }
+
+  &__name {
+    color: $Main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__product-code {
+    color: $Secondary;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 140%;
+    margin: 0 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 16px 24px;
+
+    &-name {
+      color: $Main-accent;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 140%;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16px;
+
+    &-name {
+      color: $Secondary;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 150%;
+    }
+
+    &-number {
+      color: $Main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 112.5%; /* 18px */
+    }
+  }
+
+  &__button {
+    display: block;
+    color: $white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    padding: 12px 70px;
+    height: 40px;
+    width: 166px;
```

it's not a good practice to hardcode height for elements with content inside. Just use padding
use the full parent's width
```suggestion
    width: 100%;
```